### DISCUSSION TITLE
Ignore file in subfolder.

### DISCUSSION THREAD

**Discussion by Diceble** [stryker-mutator/stryker-net#2872] [Q&A] [answered]
According to the documentation I should be able to ignore files.
I want to ignore the Program.cs.
So reading the documentation I expect I should be able to do the following
 `!**/subfoleder/Program.cs` 
However this does not give the expected result. The file keeps being scanned.

Can any one give me a little push in the right direction on what I am doing wrong? 
Many thanks

**Comment by dupdob** [comment]
IIRC, ignoring files result in ignoring all mutants in that file. I.e, mutations are still generated, but not tested.
Is it the result you get ? or are the mutations still tested ?

**Reply by Diceble** [reply]
The mutations are still getting tested. I understand that the mutations still get generated, but I expect them to get ignored and labelled as such

**Comment by rouke-broersma** [comment]
We need more information about your project structure and from which location and in which mode you're running Stryker to be able to determine why it's not working for your case. It's most likely just a matter of the path not matching.

**Reply by Diceble** [reply]
Thank you for replying.
Here some further information on my my project structure through a screenshot;

![image](https://github.com/stryker-mutator/stryker-net/assets/76655587/1fc8edd1-08e3-46da-aa8c-093e5187807c)

The file desired to ignore is in the EnduranceHub.WebAPI folder
I run stryker on the solution level where also my stryker-config.json is as you can see in the screenshot provided

**Comment by Diceble** [comment, marked as answer]
Thank you all for helping.
I found the solution (which was my own in accuracy). 
I tried to move the stryker-config.json in a seperate solution files folder.
Deciding I found that unnecessary I moved the stryker-config file back in the main folder.
Only to keep the old file alive in the solutions files folder and using that one to edit.
Sorry for the inconvenience and thanks for the people taking the time to read my question.
